Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp

```diff
@@ -220,6 +221,8 @@ void ThreadedCompositor::forceRepaint()
 
 void ThreadedCompositor::renderLayerTree()
 {
+    TraceScope traceScope(FrameCompositionStart, FrameCompositionEnd);
```

Comment:
This line isn't in a GTK/WPE-specific context, added a guard in #31759.

Reply:
Thanks @rkirsling. sorry for the inconveniences, will remember next time the compositor is used also for other ports.